content/docs/06-integrations/01-maintenance-policy.md

```diff
@@ -0,0 +1,181 @@
+
+- Container Network Interface (CNI)
+
+- Container Storage Interface (CSI)
+
+
+Add-on packs provide additional functionality that you can add to your cluster profile and are grouped as follows:
+
+- Load Balancer
+
+- Ingress
+
+- Logging
+
+- Monitoring
+
+- Security
+
+- Authenticaiton
+
+- System Apps
+
+
+Check out the [Packs List](/integrations) document, where you can use the filter buttons to display a list of Palette packs in each category and learn about the individual packs.
+
+<br />
+
+## Pack Updates
+
+Packs undergo rigorous vulnerability scans and penetration testing before they become available in Palette. The following sections describe our update schedule for each core pack category. 
+
+<br />
+
+### Kubernetes Packs
+
+We provide Cloud Native Computing Foundation (CNCF) Kubernetes updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor versions**: Provided within eight weeks of a new Kubernetes release.
+
+
+- **Patch versions**: Provided within four weeks of a new Kubernetes release.
+
+
+
+### OS Packs
+
+We provide Ubuntu LTS and CentOS updates for IaaS clusters as follows:
+
+<br />
+
+- **Major versions** - Added within 8 weeks of release.
+
+
+- **Patch and Minor versions**:  Updated at runtime using Palette’s on-demand or scheduled OS upgrades and patch-on-boot capabilities.
+
+
+### CNI Packs
+
+We provide CNI pack updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor version**: Added within 6 weeks of release.
+
+
+- **Patch versions**: Added within 4 weeks of release.
+
+
+### CSI Packs
+
+We provide CSI pack updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor version**: Added within 6 weeks of release.
+
+
+- **Patch versions**: Added within 4 weeks of release.
+
+
+### Add-on Packs
+
+We provide add-on pack updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor version**: Added within 6 weeks of release.
+
+
+- **Patch versions**: Added within 4 weeks of release.
+
+
+## Packs Deprecation
+
+
+We deprecate and remove packs when a more stable version of the pack is available or when the underlying technology becomes obsolete.
+Below are the stages of deprecation we adhere to.
+
+<br />
+
+- **Deprecated**: When a pack is deprecated, Palette displays a warning. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack.
+
+  <br />
+
+  The pack remains in *Deprecated* state for three months before it moves to *Disabled* state.
+
+
+- **Disabled**: When a pack is disabled, you will no longer be allowed to create new cluster profiles using the pack. Palette displays a message informing you to use a newer version of the pack. However, you can still deploy new clusters using existing profiles that contain the pack.
+
```

Comment:
The behavior displaying a message and prevention of creation of new cluster profiles is coming in 4.0.0. It is not there in product as of today. We can just mention our policy and document the behavior post 4.0.0